plugins/services/src/js/reducers/serviceForm/Container.js

```diff
@@ -188,6 +188,14 @@ module.exports = {
       return null;
     }
 
+    if (
+      newState.type == null &&
+      newState.docker == null &&
+      ValidatorUtil.isEmpty(newState.volumes)
```

Comment:
We decided to go with `isEmpty` just to make sure that we cover `volumes` being `undefined` and `null`